test/tables.dashdb.sql

```diff
@@ -1,40 +1,19 @@
-SET CURRENT SCHEMA = CURRENT USER;
-
-DROP TABLE LOCALUSER1;
-DROP TABLE LOCALUSER2;
-DROP TABLE LOCALUSER3;
-
-  CREATE TABLE "LOCALUSER1"
+  CREATE TABLE "?"."LOCALUSER1"
```

Comment:
These changes to make the file parameterized kind of change the file type, don't they? It's not _really_ a SQL script file (which one would expect to be directly executable), it's more like a template for a SQL script, isn't it?